frontend/src/metabase/App.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { Component, PropTypes } from "react";
+
+import Navbar from "metabase/nav/containers/Navbar.jsx";
+
+export default class App extends Component {
```

Comment:
At one point I was using the lifecycle methods. It's moderately likely we might need to in the future, and there's only one instance of this so memory consumption isn't a concern, so I'll probably leave it as-is for now.